src/app/body/create-new-organization/team-details/team-details.component.ts

```diff
@@ -17,13 +20,27 @@ export class TeamDetailsComponent implements OnInit {
 
   childStep: number = 1
 
-  constructor(private functions: AngularFireFunctions, public validationService: ValidationService, private router: Router) { }
-
-  ngOnInit(): void { }
-
-
+  constructor(private functions: AngularFireFunctions, public validationService: ValidationService, private router: Router, public orgTeamService: OrgTeamService, public backendService: BackendService) { }
+
+  ngOnInit(): void {
+      try{
+        this.orgTeamService.getTeamDetails().subscribe(
+        data =>{
+          this.teamName = data.TeamName;
+          this.teamId = data.TeamId;
+          this.teamDescription = data.TeamDescription;
+          this.teamManagerEmail = data.TeamManagerEmail;
+          this.teamMembers = data.TeamMembers.join(',');
+      }, error=>{
+          throw error;
+        }
+    );}catch(error){
+      console.log("Team does not exist");
+    }
```

Comment:
This is not needed every-time, only when we want to update the team details we need it, else we should keep the fields empty for to create the new team.